zpy/objects.py

```diff
@@ -586,3 +586,83 @@ def restore_pose(
     obj = verify(obj)
     log.info(f"Restoring pose {pose_name} to object {obj.name}")
     obj.matrix_world = _SAVED_POSES[pose_name]
+
+
+def lighting_randomize(
+    path: Union[Path, str] = "hdri",
+    lighting_styles: List = ["bounded", "sun", "hdri"],
+    energy_jitter: bool = True,
+    lighting_styles_weights: List[int] = [3, 5, 2],
+) -> None:
+    """Randomizes Lighting Types.
```

Comment:
This isn't exactly the docstring format we have. Look into an addon such as this one, which will basically allow you to hit one shortcut to create a template docstring.
![image](https://user-images.githubusercontent.com/8493427/116941712-11f57b00-ac25-11eb-807b-c0260420d524.png)
![image](https://user-images.githubusercontent.com/8493427/116941872-60a31500-ac25-11eb-91bd-fdbe447b3788.png)